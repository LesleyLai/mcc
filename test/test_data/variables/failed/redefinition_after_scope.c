int main(void)
{
  int x = 42;
  {
    x = 42;
  }
  int x = 2;
  return x;
}
