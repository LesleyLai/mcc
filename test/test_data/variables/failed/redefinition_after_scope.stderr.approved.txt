{{base_dir}}/variables/failed/redefinition_after_scope.c:7:7: Error: redefinition of 'x'
7 |   int x = 2;
  |       ^

{{base_dir}}/variables/failed/redefinition_after_scope.c:8:10: Error: Expect end of the file
8 |   return x;
  |          ^

